doc/configuration.rst

```diff
@@ -32,16 +38,55 @@ configuration file. For instance if you have a Task definition:
         date = luigi.DateParameter(default=datetime.date.today())
         # ...
 
-Then you can override the default value for date by providing it in the
-configuration:
+Then you can override the default value for ``DailyReport().date`` by providing
+it in the configuration:
 
 ::
 
     [DailyReport]
     date: 2012-01-01
 
-You can also use ``config_path`` as an argument to the ``Parameter`` if
-you want to use a specific section in the config.
+.. _ConfigClasses:
+
+Configuration classes
```

Comment:
Being able to hyperlink to this documentation is going to save me sooooo much time. :)